run/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from storage.models import File
+from game_runner import settings
+from game.models import OperationParameter, Game, Operation
+import uuid
+
+
+class ParameterValue(models.Model):
+    _value = models.TextField()
+    run = models.ForeignKey('Run', related_name='parameter_value_set')
+    parameter = models.ForeignKey(OperationParameter)
+
+    @property
```

Comment:
You should define setter as well. So if the parameter is a file, we probably would want to use parameter_value.value = my_file.